### ISSUE TITLE
Scrolling does not work correctly after tab through children

### ISSUE THREAD

**Issue by aarowman** [sag1v/react-elastic-carousel#38]
**Describe the bug**
If you use tabs to navigate to the children, then the pages are unaware of the change and scroll is incorrect.

**To Reproduce**
1. Set the children to a list of elements larger than one page (e.g. size 5 page with 20 elements)
2. run app and tab to the elements (e.g. keep tabbing to the 6th element or more) WITHOUT clicking the nav buttons - this will cause the carousel to "scroll" one item at a time
3. click Prev button
Then, the prev does not scroll, because it thinks it is still on the first page. This behaves the same for a final page or other pages.

**Expected behavior**
The scroll buttons continue to scroll until reaching the true end. 
Ideally the ref state would be updated to reflect the current page/offset when tabbing, or that tabs would not cause scrolling, but instead would tab to the next button.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Firefox
 - Version: 60.9.0esr

**Smartphone (please complete the following information):**
N/A

**Additional context**
I know that the goTo(), next/prev arrows, and pagination buttons all trigger a state change for the active page, so the scrolls are likely tied to that.. and the carousel is actually rendering the entire list, but hiding all elements besides the current "page", which allows scrolling to work easier (css: overflow: hidden). This causes a few weird quirks when you 'scroll' without using the library functions to do it (e.g. tabs)

**Comment by sag1v** [comment]
Thanks! Unfortunately this component is not a11y oriented. yet!
For this specific case, i would say we can pass `tabIndex={0}` to the arrows and all child wrappers that are visible. This should cover the bug isn't it?

**Comment by aarowman** [comment]
@sag1v If you can give a tabIndex only to the visible children, then NOT on the hidden ones, I do think that would do it. 

Similarly, I think the current situation is actually ok from an a11y view. I believe tabs will go to all content and then skip the arrows, which is satisfactory.

Doing only tabs or only clicks is fine. The difficulty is when combining tabs with clicks.

**Comment by sag1v** [comment]
@aarowman Done. you can now navigate with Tab through arrow buttons, visible items and dots :)

**Comment by aarowman** [comment]
Awesome! I look forward to trying it out
